package.json

```diff
@@ -60,6 +60,7 @@
     "moment-timezone": "^0.5.33",
     "polished": "^4.1.1",
     "react-activestorage-provider": "^0.8.0",
+    "react-modal": "^3.14.3",
```

Comment:
Do we really need the extra component? A modal does nothing that our right-panel doesn't already do in terms of overlays